src/modules/Contributors/index.jsx

```diff
@@ -16,58 +17,109 @@ export default class Contributors extends Component {
     filename: PropTypes.string.isRequired,
   }
 
+  static getGithubUrl(repo, filename, action) {
+    return `${repo}/${action}/master/content/${filename}`
+  }
+
   render() {
     const filename = this.props.filename
+    const pkg = this.context.pkg
```

Comment:
indeed. When I wrote that in the first place, I didn't know about it yet.
Will do everywhere :)